test/e2e/kuma_test.go

```diff
@@ -77,6 +78,11 @@ func TestDeployAllInOneDBLESSKuma(t *testing.T) {
 	t.Log("running ingress tests to verify all-in-one deployed ingress controller and proxy are functional")
 	deployIngress(ctx, t, env)
 	service, err := env.Cluster().Client().CoreV1().Services("default").Get(ctx, "httpbin", metav1.GetOptions{})
+
+	t.Logf("service %#v", service)
```

Comment:
This might get quite verbose. I wonder if we couldn't make this a bit prettier, using some sort of pretty printer for structs? 🤔

Reply:
`json.MarshalIdent` could be used, but I think printing in `#v` is clear enough.